PrimeroAdministrationandConfigurationGuide.md

```diff
@@ -546,6 +583,10 @@ Once a Lookup is created, you can add to a field type and select whether you can
 
 To edit the properties of an existing Lookup, find the name and click _Edit_ in the options in the right column. From the _Show_ page, click the EDIT button at the top of the screen.
 
+## Jordan Case Identity
```

Comment:
See comment for the same section on the 1.4 PR. Whatever you end up doing with that one, put it in here.